modules/end-user-guide/partials/proc_creating-a-new-workspace-from-the-dashboard.adoc

```diff
@@ -17,22 +17,27 @@ To edit the devfile:
 
 . In the *Workspaces* window, click the btn:[Add Workspace] button. The *Custom Workspace* page should be opened.
 
-. Scroll down to the *Devfile* section. In the *Devfile editor*, add necessary changes.
+. Scroll down to the *Devfile* section and edit the devfile using *Devfile editor*.
 +
 image::workspaces/minimal-devfile.png[Devfile editor with the minimal devfile, link="../_images/workspaces/minimal-devfile.png"]
 +
-[NOTE]
-.Example: add a project
-====
+For an example demonstration, see below.
++
+[id="adding-a-git-project-into-a-workspace-using-a-devfile_{context}"]
+.Example - Adding a `.git` project into a workspace using a devfile
++
+In the following example, the project `{project-context}` acts as the example of a user's project. A user specifies this project using the `name` attribute of a devfile. The `location` attribute defines the source repository represented by an URL to a Git repository or ZIP archive.
```

Comment:
"In the example... acts as an example". Suggest dropping one or rewording to smth like "In this section..."

Reply:
I changed it to  "instance"